src/effects/VST/VSTEffect.cpp

```diff
@@ -1154,6 +1165,7 @@ int VSTEffect::ShowClientInterface(
    //   mProcessLevel = 1;      // in GUI thread
 
    VSTEffectValidator* vstValidator = static_cast<VSTEffectValidator*>(validator);
+   mValidator = vstValidator;
```

Comment:
Step 5.1 adds this assignment, but never uses the member, and never reassigns it back to null.  But I'm still progressing through the 5.x commits, which may be squashed into one if I approve them.

Reply:
Correct. I introduced `mValidator` as a way to call `callSetParameter` on the Validator's instance, in `::OnSlider`, as you previously suggested (or so I understood). Then I found out that things work also without that call (and I left the code there commented out) so I could delete `mValidator` now. Shall I do it?